src/Device/PermissionsContext.tsx

```diff
@@ -46,12 +43,14 @@ export type ENPermissionStatus =
   | "Active"
   | "Disabled"
   | "BluetoothOff"
+  | "LocationOffAndRequired"
```

Comment:
@mattThousand, changed the name of this because it clarifies that `LocationOffAndNotRequired` is not a possible state for `ENPermissionStatus`. We may want to change this on the native layer in some places, too?

Reply:
@juancruzgs This also may be relevant to the work you are doing tomorrow ^